src/OpenLoco/src/Ui/Windows/Construction/Construction.h

```diff
@@ -9,6 +10,7 @@
 #include "Ui/Widgets/TabWidget.h"
 #include "Ui/Widgets/Wt3Widget.h"
 #include "Ui/WindowManager.h"
+#include <GameCommands/Track/CreateTrackMod.h>
```

Comment:
This is coming from the same 'project', so I'd prefer we use:
```suggestion
#include "GameCommands/Track/CreateTrackMod.h"
```

Reply:
ah of course, this would have been VS  auto-adding the header :( fixed now